kevm-pyk/src/tests/integration/test_conformance.py

```diff
@@ -47,7 +51,23 @@ def _test(gst_file: Path, schedule: str, mode: str, usegas: bool) -> None:
         if test_name in skipped_gst_tests:
             continue
         res = interpret({test_name: test}, schedule, mode, chainid, usegas, check=False)
-        _assert_exit_code_zero(res)
+
+        try:
+            _assert_exit_code_zero(res)
+        except AssertionError as _exception:
+            if not save_failing:
+                raise _exception
+            failing_tests.append(test_name)
+
+    if failing_tests:
+        if save_failing:
+            with FAILING_TESTS_FILE.open('a') as ff:
+                if len(failing_tests) == len(gst_data):
+                    ff.write(f'{gst_file_relative_path},*\n')
```

Comment:
What's the purpose of this special case?

Reply:
If all the tests in a GST file are failing, mark the entire file as failing using the `file_name,*` syntax instead of marking each test individually.